fili-core/src/main/java/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMaker.java

```diff
@@ -39,18 +42,36 @@
      * @param metricDictionary  The dictionary used to resolve dependent metrics when building the LogicalMetric
      * @param function  The arithmetic operation performed by the LogicalMetrics constructed by this maker
      * @param resultSetMapperSupplier  A function that takes a metric column name and produces at build time, a result
+     * @param defaultProtocolSupport Protocols to support by default
```

Comment:
Thing this should actually go one line lower. `set mapper` just below goes with `resultSetMapperSupplier`.

Reply:
ArithmeticMaker uses the protocolSupport as a base and then subtracts the blacklists from the dependencies before attaching it to a new metric.

Reply:
Renaming to 'baseProtocolSupport' and clarifying in param